website/src/documentation/sdks/nexmark.md

```diff
@@ -171,18 +173,24 @@ Number of events generators:
 
     --numEventGenerators=4
 
-Run query N:
+Queries can be run by their name or by their number (number is still there for backward compatibility, only the queries 0 to 12 have a number)
+
+Run query **N**:
 
     --query=N
 
+Run query called **PASSTROUGH**:
+
+    --query=PASSTROUGH
+
 ### Available Suites
 The suite to run can be chosen using this configuration parameter:
 
     --suite=SUITE
 
 Available suites are:
 * DEFAULT: Test default configuration with query 0.
-* SMOKE: Run the 12 default configurations.
+* SMOKE: Run the 14 queries with the default configuration.
```

Comment:
Maybe just "all queries" to make it future-proof :-)

Reply:
sure :) thx